Makefile

```diff
@@ -62,10 +62,10 @@ libtree-sitter-$(PARSER_NAME).$(SOEXTVER): $(OBJ)
 	ln -sf $@ libtree-sitter-$(PARSER_NAME).$(SOEXT)
 	ln -sf $@ libtree-sitter-$(PARSER_NAME).$(SOEXTVER_MAJOR)
 
- bindings/c/tree-sitter.h:
+bindings/c/tree-sitter-$(PARSER_NAME).h:
 	sed -e 's|@UPPER_PARSERNAME@|$(UPPER_PARSER_NAME)|' \
 		-e 's|@PARSERNAME@|$(PARSER_NAME)|' \
-		$@.in > $@
+		bindings/c/tree-sitter.in > $@
```

Comment:
I assume this should be `bindings/c/tree-sitter.h.in > $@` (when I try this locally, I don't have just a `tree-sitter.in`).

Reply:
it totally should be! thanks